.github/workflows/benchmark.yml

```
@@ -17,31 +17,15 @@ jobs:
         os: [ubuntu-latest, macos-latest]
 
     steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 2
-
-      - uses: ./.github/actions/setup-deps
+      - uses: actions/checkout@v
```

```suggestion
      - uses: actions/checkout@v3
```

and you want to remove fetch-depth?